### ISSUE TITLE
Network unreachable in container

### ISSUE THREAD

**Issue by cwl233** [kubeedge/kubeedge#1364]
**What happened**:

I deployed my deployment, but kubeedge ran my container first, then the kubeedge/pause:3.1 container.
So my container can not access the network.

This problem does not happen every time.
When it happens, the "docker ps" command show like this:
![image](https://user-images.githubusercontent.com/50825450/71726320-b7ea7700-2e71-11ea-8c84-5ddfb7dda122.png)

And in this case, I would get "Network unreachable" in my container.
![image](https://user-images.githubusercontent.com/50825450/71726738-f2a0df00-2e72-11ea-8719-ef93e3af57a9.png)

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- KubeEdge version(e.g. `cloudcore/edgecore --version`):  

**CloudSide Environment**:
- Hardware configuration (e.g. `lscpu`):
- OS (e.g. `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Go version (e.g. `go version`): 
- Others:

**EdgeSide Environment**:
- edgecore version (e.g. `edgecore --version`):
- Hardware configuration (e.g. `lscpu`):
- OS (e.g. `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Go version (e.g. `go version`): 
- Others:

**Comment by faicker** [comment]
the same problem.

**Comment by fisherxu** [comment]
@faicker Could you help clarify how to reproduce this issue? I test many times in my host, and haven't seen the issue... The pod network set to `hostNetwork` or `portMapping`?

**Comment by faicker** [comment]
example yaml,
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
    name: nginx-2
    labels:
        app: nginx

spec:
    replicas: 1
    selector:
        matchLabels:
            app: nginx
    template:
        metadata:
            labels:
                app: nginx
        spec:
            containers:
            - name: nginx
              image: nginx:1.17.6
              ports:
              - containerPort: 80
                hostPort: 82
````
try some times.

**Comment by faicker** [comment]
It seems to be solved in v1.4.0
